core/templates/pages/learner-dashboard-page/learner-groups-tab.component.css

```diff
@@ -236,12 +259,30 @@
 
   .learner-groups-tab .oppia-learner-group-summary-card {
     align-items: unset;
-    flex-direction: column;
     gap: 10px;
   }
 
   .learner-groups-tab .learner-group-view-details-button {
     font-size: 14px;
+    justify-content: flex-end;
     margin-right: 0;
   }
+
+  .learner-groups-tab .oppia-learner-group-summary-card-cta-buttons {
+    align-items: center;
+    display: inline-grid;
+    gap: 11px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .learner-groups-tab .learner-of-learner-groups-section {
+    margin-left: 12px;
+  }
+}
+@media (max-width: 767px) {
+  .learner-groups-tab .learner-groups-section-heading {
+    font-family: Capriola,Roboto,Arial,sans-serif;
```

Comment:
Not done. As part of my previous review, I had required the spacing here be fixed.

Reply:
Done, sorry I missed it